web/src/lib/services/shared-link.service.ts

```diff
@@ -29,7 +29,7 @@ export const getSharedLinkActions = ($t: MessageFormatter, sharedLink: SharedLin
 
   const Delete: ActionItem = {
     title: $t('delete_link'),
-    icon: mdiTrashCanOutline,
+    icon: mdiDeleteOutline,
```

Comment:
We use the trash can outline everywhere, why change it here and make it inconsistent?

Reply:
I will open another PR to change at other places